tools/shared/src/main/scala/org/scalajs/core/tools/optimizer/Analysis.scala

```diff
@@ -76,6 +76,7 @@ object Analysis {
     // TODO Get rid of InheritedConstructor when we can break binary compat
     final case object InheritedConstructor extends MethodSyntheticKind
     final case class ReflectiveProxy(target: String) extends MethodSyntheticKind
+    final case class DefaultBridge(targetInterface: String) extends MethodSyntheticKind
```

Comment:
This might actually be a good spot to document the shapes of all these synthetic methods (in a later commit / PR). Just had that thought.